Flepper.QueryBuilder/SqlColumn.cs

```diff
@@ -31,49 +31,12 @@ public class SqlColumn
         /// </summary>
         public string TableAlias { get; }
 
+
         internal SqlColumn(string column)
         {
             if (IsNullOrWhiteSpace(column)) throw new ArgumentNullException($"{nameof(column)} cannot be null or empty");
 
-            TableAlias = GetTableAlias(column);
-
-            var containsAlias = ContainsAlias(column);
-            var containsTableAlias = ContainsTableAlias(column);
-
-            if (containsAlias && containsTableAlias)
-            {
-                var columnAsAlias = default(string);
-
-                (TableAlias, columnAsAlias) = column.Split(TableAliasSplitter, StringSplitOptions.RemoveEmptyEntries);
-                (Column, Alias) = columnAsAlias.Split(AliasSplitter, StringSplitOptions.RemoveEmptyEntries); ;
-            }
-            else if (containsAlias)
-            {
-                (Column, Alias) = column.Split(AliasSplitter, StringSplitOptions.RemoveEmptyEntries);
-            }
-            else if (containsTableAlias)
-            {
-                (TableAlias, Column) = column.Split(TableAliasSplitter, StringSplitOptions.RemoveEmptyEntries);
-            }
-            else
-            {
-                Column = $"{column}";
-            }
-
-            if(Column != "*")
-            {
-                if (!Column.StartsWith("[")) Column = $"[{Column}";
-                if(!Column.EndsWith("]")) Column = $"{Column}]";
-            }
-
-            if(TableAlias != null)
-            {
-                if (!TableAlias.StartsWith("[")) TableAlias = $"[{TableAlias}";
-                if (!TableAlias.EndsWith("]")) TableAlias = $"{TableAlias}]";
-            }
-
-            if (!IsNullOrWhiteSpace(Alias) && !Column.Contains(ALIAS)) Column = $"{Column} AS {Alias.Trim()}";
-            if (!IsNullOrWhiteSpace(TableAlias) && !Column.Contains(TABLE_ALIAS)) Column = $"{TableAlias}.{Column}";
+            Column = column == "*" ? $"{column}" : $"[{column}]";
```

Comment:
Does not need to use interpolation in the true expression, replace for:

`Column = column == "*" ? column : $"[{column}]"; `